qa/workunits/fs/snaps/snaptest-name-limits.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+#
+# This tests snapshot names limits: names have to be < 240 chars
+#
+
+function cleanup ()
+{
+	rmdir d1/.snap/*
+	rm -rf d1
+}
+
+function fail ()
+{
+	echo $@
+	cleanup
+	exit 1
+}
+
+mkdir d1
+
+longname=$(printf "%.241d" 2)
+mkdir d1/.snap/$longname 2> /dev/null
```

Comment:
`.snap` doesn't exist here. This needs `mkdir -p` to work correctly or create it earlier.

Reply:
Yeah, `.snap` is special here (see [1]).  You don't need to create it as it exists in every cephfs directory.  That's one of the non-POSIX things in cephfs.

[1] https://docs.ceph.com/en/pacific/dev/cephfs-snapshots/